pkg/checks/daemonSet/daemonSet.go

```diff
@@ -151,6 +156,36 @@ func (dsc *Checker) Shutdown() error {
 
 }
 
+// FindAllUniqueTaints returns a list of all taints present on any node group in the cluster
```

Comment:
This name is a little misleading.  It says that it finds unique taints, but it returns a slice of tolerations